gulpfile.js

```diff
@@ -24,6 +24,9 @@ var distPath  = themePath + '/dist';
 var assetPath = themePath + '/assets';
 var revision = Math.floor(Date.now() / 1000);
 
+// CDN path replacing
+var fullDistPath = '/' + distPath;
```

Comment:
I prefer `distPathAbsolute` wdyt? Also no need for the comment and extra space, you can put under line 23 or 24.